### DISCUSSION TITLE
unmet dependencies when installing gui: libnode72, nodejs

### DISCUSSION THREAD

**Discussion by darkdarkgray** [Chia-Network/chia-blockchain#9673] [Support]
I have an issue with gui installation.
After I run ./install-gui.sh I get this:
```
The following packages have unmet dependencies:
 libnode72 : Conflicts: nodejs-legacy
 nodejs : Conflicts: npm
```

**Comment by darkdarkgray** [comment]
```
sudo apt remove nodejs
sudo rm -rf /usr/local/bin/node*
sudo rm -rf /usr/local/bin/npm*
sudo rm -rf /etc/apt/sources.list.d/nodesource.list
```
These commands solved the situation.

**Reply by edavalosanaya** [reply]
Thank you! This solved my issue.

**Comment by Taweesakpintakam7m1** [comment]
> I have an issue with gui installation. After I run ./install-gui.sh I get this:
> 
> ```
> The following packages have unmet dependencies:
>  libnode72 : Conflicts: nodejs-legacy
>  nodejs : Conflicts: npm
> ```
